man/hbv.Rd

```diff
@@ -0,0 +1,135 @@
+% Generated by roxygen2: do not edit by hand
+% Please edit documentation in R/hbv.R
+\name{hbv}
+\alias{hbv}
+\alias{hbv.sim}
+\alias{hbvrouting}
+\alias{hbvrouting.sim}
+\alias{hbv.ranges}
+\alias{hbvrouting.ranges}
+\title{HBV rainfall-runoff model}
+\usage{
+hbv.sim(
+  DATA,
+  tt,
+  cfmax,
+  sfcf,
+  cwh,
+  cfr,
+  fc,
+  lp,
+  beta,
+  k0,
+  k1,
+  k2,
+  uzl,
+  perc,
+  return_state = FALSE
+)
+
+hbvrouting.sim(U, maxbas, epsilon = hydromad.getOption("sim.epsilon"))
+
+hbv.ranges()
+
+hbvrouting.ranges()
+}
+\arguments{
+\item{DATA}{A time-series like object with columns P (precipitation in mm),
+E (potential evapotranspiration in mm), T (average air temperature in ºC).}
```

Comment:
The separation of E and T is new in hydromad. I think it makes sense but we'll need to think about how this affects, e.g. snow.sim before the 1.0 release